spine-c/src/spine/Bone.c

```diff
@@ -209,9 +206,9 @@ float spBone_getWorldScaleY (spBone* self) {
 }
 
 void spBone_worldToLocal (spBone* self, float worldX, float worldY, float* localX, float* localY) {
-	float x = worldX - self->worldX, y = worldY - self->worldY;
 	float a = self->a, b = self->b, c = self->c, d = self->d;
 	float invDet = 1 / (a * d - b * c);
+	float x = worldX - self->worldX, y = worldY - self->worldY;
 	*localX = (x * d * invDet - y * b * invDet);
 	*localY = (y * a * invDet - x * c * invDet);
 }
```

Comment:
Yeah, `rotateWorld` is not used by the runtime (or editor) but is useful to users. The others are there for the same reason, though may not be useful as often. We're trying to save users from having to deal with much math.